### PR TITLE
Show next waypoints as POI in the hud + hud tweaks

### PR DESCRIPTION
**Pull request by OlivierC-FR**
Show the next waypoints in the hud.
To enable, enter the count of next waypoints in the CMS menu :
OSD > Hub > Displayed items > Next waypoints : (from 0 to 3, 0 means disabled)
Or CLI command: set osd_hud_wp_disp = 0 to 3

For now, require one of these OSD fonts : https://www.dropbox.com/s/v4yd3bpraty2h65/visions.zip?dl=1

Then start a WP mission, the next WPs will appear **as soon as the mission is loaded in FC memory AND that the aircraft has a valid GPS heading (so it must move if it's not fitted with a magnetometer)**

----------------------
+ Fix for the AH mode displayed instead of WP
+ Less hud Y margins (because NTSC only 13 lines)
+ Improved side-stacking for the POIs
+ Smaller min range for radar POIs (3m)

### CODE REVIEW

**Comment by OlivierC-FR** [comment]
Sample (old version, outdated graphics) : https://www.youtube.com/watch?v=CqKNGY4pogU
Newer graphics : https://drive.google.com/file/d/1YQbU1LiH9IS7nMToIKziXBvbSRDEFKK8/view

**Comment by OlivierC-FR** [comment]
https://github.com/iNavFlight/inav-configurator/pull/900

**Comment by danarrib** [comment]
I tested this PR on both MR and FW, it's working fine.
Would be nice if it's included on the 2.4 release.

File: src/main/build/version.h [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 
 #define FC_VERSION_MAJOR            2  // increment when a major release is made (big new feature, etc)
-#define FC_VERSION_MINOR            4  // increment when a minor release is made (small new feature, change etc)
+#define FC_VERSION_MINOR            3  // increment when a minor release is made (small new feature, change etc)
```
Comment (danarrib) [inline src/main/build/version.h]: Your change should not mess with INAV version. There is another PR just to bump the firmware version.

Comment (DzikuVx) [reply src/main/build/version.h]: @OlivierC-FR could you fix that maybe?

Comment (OlivierC-FR) [reply src/main/build/version.h]: done gentlemen !

**Comment by DzikuVx** [comment]
@OlivierC-FR @danarrib you think we can merge it now?

**Comment by danarrib** [comment]
> @OlivierC-FR @danarrib you think we can merge it now?

I think we can. I'm flying it, everything is working fine.

**Comment by OlivierC-FR** [comment]
Also been flying it a long time, just there's some kind of "limitation" if we can call it like that, when the mission is finished, on the OSD at the top of the hud area remains displayed the counter with current and max WP, like "WP 8/8" until you restart the FC, or until you restart the mission. Thats because to keep the code simple and light I didn't bother to erase that with a condition. I don't think that's worth it anyway, and it's not intrusive.

**Comment by DzikuVx** [comment]
Let's merge it then!

**Pull request merged**